### ISSUE TITLE
How to extract audio and video alignment embeddings?

### ISSUE THREAD

**Issue by tlemangen** [vskadandale/vocalist#20]
Which step in the forward method of SyncTransformer can get the audio and video alignment embedding?

```python
def forward(self, frame_seq, mel_seq):
        B = frame_seq.shape[0]
        vid_embedding = self.vid_prenet(frame_seq.view(B,-1,3,48,96).permute(0,2,3,4,1).contiguous())
        aud_embedding = self.aud_prenet(mel_seq)

        vid_embedding = vid_embedding.squeeze(2).squeeze(2)
        aud_embedding = aud_embedding.squeeze(2)

        vid_embedding = vid_embedding.permute(2, 0, 1).contiguous()
        aud_embedding = aud_embedding.permute(2, 0, 1).contiguous()

        av_embedding = self.av_transformer(aud_embedding, vid_embedding, vid_embedding)
        va_embedding = self.va_transformer(vid_embedding, aud_embedding, aud_embedding)

        tranformer_out = self.mem_transformer(av_embedding, va_embedding, va_embedding)
        t = av_embedding.shape[0]

        out = F.max_pool1d(tranformer_out.permute(1, 2, 0).contiguous(), t).squeeze(-1)
        h_pooled = self.activ1(self.fc(out))  # [batch_size, d_model]
        logits_clsf = (self.classifier(h_pooled))
        return logits_clsf.squeeze(-1)
```

I think `vid_embedding` and `aud_embedding` may be aligned embeddings, but the measured alignment and non-alignment accuracy is not high, about 50%. This is almost a random guess. Where did I go wrong?
